tests/app/test_info.py

```diff
@@ -0,0 +1,15 @@
+import unittest
```

Comment:
In another file we're using pytest. Do we want to use both?

Reply:
I can't remember if you need Unittest to get pytest to work or not... Will need to check

Reply:
No I believe they are 2 separate modules both independent. We just need to decide which one to go with. But i guess we can decide later when we write more code